src/components/Audience.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import axios from 'axios';
+
+class Audience extends React.Component {
+    constructor (props){
+        super(props);
+        this.state={
+            users:[],
+            user: {
+                id: 1,
+                email: '',
+                reviews: [],
+                group: '',
+                groupMembers: []
+            },
+            value:'',
+            suggestions:[]
+        }
+    }
+    componentDidMount(){
+        const user={
+            email: 'vlad@aviata.me',
+            reviews: [{email:'ksenya.v@aviata.kz'},{email:'alena@aviata.me'},{email:'dauren@chocotravel.com'},{email:'david@chocotravel.com'}],
+            group: 'top',
+            groupMembers:[{email: 'kanat@aviata.me'},{email:'shaken@chocotravel.com'},{email:'ayuna@aviata.me'}]
+        }
+        this.setState({
+            user:user
+        })
+        axios.get('http://46.101.246.71:8000/users/')
+            .then(response => {
+                if (response.data.length > 0) {
+                this.setState({
+                    users: response.data.map(user => user.email),
+                })
+                console.log(this.state.users)
+                }
+            })
+            .catch((error) => {
+                console.log(error);
+            })
+        this.state.user.reviews.forEach(item=>console.log(item))
+    }
+    onAutoChange = (e) =>{
+        this.items=this.state.users;
+
+        const value=e.target.value;
+        let suggestions=[];
+        if(value.length>0){
+            const regex=new RegExp(`^${value}`,'i');
+            suggestions=this.items.sort().filter(v=>regex.test(v));
+        }
+
+        this.setState(()=>({suggestions, value:value}));
+        
+    }
+    renderSuggestions(){
+        const {suggestions}=this.state;
```

Comment:
```suggestion
        const {suggestions} = this.state;
```
много где не хватает пробелов